### DISCUSSION TITLE
Nuxt3 NuxtLayout generating "Failed to resolve component: Nuxt"

### DISCUSSION THREAD

**Discussion by thehoegh** [nuxt/nuxt#17769] [Questions] [answered]
Hi, 

I'm playing with Nuxt3, trying to do a super simple MWE of using the  `app.vue`, `pages/index.vue` and `layouts/default.vue`, but keep smashing into a wall with default layouts. 

layouts/default.vue
```
<template>
    <div>
        <h1>Default Layout</h1>
        <Nuxt />
    </div>
</template>

<script>
export default {
}
</script>
```

pages/index.vue
```
<template>
    <div>
        The index page
    </div>
</template>

<script>
export default {
}
</script>
```

app.vue, according to https://github.com/nuxt/framework/pull/3011
```
<template>
  <div>
    <NuxtLayout>
      <NuxtPage />
    </NuxtLayout>
  </div>
</template>
```

If I comment out the `<Nuxt />` from `layouts/default.vue`, the `h1` renders as expected, and without the content of `index.vue` obviously. However, _with_ `<Nuxt />`, I get the error `[Vue warn]: Failed to resolve component: Nuxt` generating a massive 500 error. 

What am I doing wrong?

<img width="757" alt="image" src="https://user-images.githubusercontent.com/4199918/159114141-7a771ecd-84f5-44b0-add9-9a1921df4d29.png">

**Comment by humandetail** [comment, marked as answer]
Did you mean `<slot />`?

**Reply by thehoegh** [reply]
I was about to say "no", but it may appear that I've literally have mixed `v3.nuxtjs.org/docs/` and `nuxtjs.org/docs`. Wow. Too simple solution, as usual. Thanks 🙏
